pkg/cli/config/server_context.go

```diff
@@ -26,6 +26,9 @@ type ServerContext struct {
 	Server   string `json:"server" survey:"server"`
 	Token    string `json:"token" survey:"token"`
 	Insecure bool   `json:"insecure" survey:"insecure"`
+
+	// Reachable.
+	Reachable bool
```

Comment:
can we use `Unreachable bool` here?

Reply:
Prefer positive and affirmative variable names, Reachable might be more intuitive.